src/app/organization/holidays/view-holidays/view-holidays.component.html

```diff
@@ -0,0 +1,69 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" (click)="activateHoliday()" *ngIf="holidayData.status.value !== 'Active'">
+    Activate
+  </button>
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn" (click)="deleteHoliday()">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="header">
+          Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.name }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          From Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.fromDate | date:'yyyy-MM-dd' }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          To Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.toDate | date:'yyyy-MM-dd' }}
+        </div>
+
+        <div fxFlex="50%" class="header" *ngIf="holidayData.repaymentsRescheduledTo !== undefined || holidayData.repaymentsRescheduledTo !== null">
+          Repayments Scheduled To
+        </div>
+
+        <div fxFlex="50%" *ngIf="holidayData.repaymentsRescheduledTo !== undefined || holidayData.repaymentsRescheduledTo !== null">
+          {{ holidayData.repaymentsRescheduledTo | date:'yyyy-MM-dd' }}
+        </div>
+
+        <div fxFlex="50%" class="header" *ngIf="holidayData.repaymentsRescheduledTo === undefined || holidayData.repaymentsRescheduledTo === null">
+          Repayments Scheduled To
+        </div>
+
+        <div fxFlex="50%" *ngIf="holidayData.repaymentsRescheduledTo === undefined || holidayData.repaymentsRescheduledTo === null">
+          {{ holidayData.nextrepaymentdate | date:'yyyy-MM-dd' }}
```

Comment:
`nextrepaymentdate` isn't any key in the object. You just have to display as `Next Repayment Date` .

Reply:
You mean the heading. Not the value

Reply:
Yes, Have a cross-check from the community app code.

Reply:
ok